tests/Bandits/BanditReferenceIndexerTest.php

```diff
@@ -148,7 +149,36 @@ public function testBadBanditVariation(): void
 
         $this->expectException(InvalidConfigurationException::class);
 
-        BanditReferenceIndexer::from(self::$banditReferences);
+        BanditReferenceIndexer::from($refsToLoad);
+    }
+
+    public function testGetBanditModelsEmptyReferences()
+    {
+        $indexer = BanditReferenceIndexer::empty();
+        $this->assertEquals([], $indexer->getBanditModelVersionReferences());
+    }
+
+    public function testGetBanditModelsNoVariations()
```

Comment:
Oh, interesting! I don't think this is possible to be generated by the UFC at the moment, but the data model allows it so appreciate the defensiveness in checking this.